ixmp/core.py

```diff
@@ -1150,16 +1172,18 @@ def clone(self, model=None, scenario=None, annotation=None,
         scenario : str, optional
             New scenario name. If not given, use the existing scenario name.
         annotation : str, optional
-            Explanatory comment for the clone operation.
-        keep_solution : bool
-            If :py:const:`True`, include data from an existing model solution
-            in the clone.
-        first_model_year: int
-            If given, all time series data in the Scenario is omitted from the
-            clone for years from `first_model_year` onwards. Time series data
+            Explanatory comment for the clone commit message to the database.
+        keep_solution : bool, default True
```

Comment:
[The convention](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard) is to do like `keep_solution : bool, optional`. The default is shown in the built docs automatically; [see here](https://message.iiasa.ac.at/projects/ixmp/en/latest/api/python_core.html#ixmp.Scenario.clone).

Reply:
done